samples/Dantooine/Dantooine.Client/Client/Services/AuthorizedHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Dantooine.BFF.Client.Services
+{
+    // orig src https://github.com/berhir/BlazorWebAssemblyCookieAuth
+    public class AuthorizedHandler : DelegatingHandler
```

Comment:
To avoid maintaining this class, could we instead just do a try/catch in the Razor page and call `SignIn` in the page itself?

Reply:
I think this should be kept